big5/THIRD-PARTY.txt

```diff
@@ -0,0 +1,210 @@
+The OSB big5 workload includes the following third-party software:
+
+** Elasticsearch vs. OpenSearch comparison benchmark (https://github.com/elastic/elasticsearch-opensearch-benchmark)
```

Comment:
did you intend to make this line bold ? in that case you will need two more ** after the line for it to render

Reply:
Thanks for pointing it out.  This is intentional; it was lifted from a recommendation guide for open-source compliance, which uses this double-asterisk prefix for some reason.